tasks/install_odl.yml

```diff
@@ -1,5 +1,23 @@
 ---
+- set_fact:
+    os="{{ ansible_distribution }}_{{ ansible_distribution_major_version }}.0"
```

Comment:
I have added some docs to help understand the variables better. Let me know if things are still not clear.